content/notes/concepts.md

```diff
@@ -35,87 +36,64 @@ Different model types can vary across these dimensions. Choosing a model type wi
 
 The diagram below shows the different options available for each building block. We will describe these below. Note that even these options are themselves broad umbrellas and can themselves branch into further variations when developed further.
```

Comment:
broad umbrellas => broad categories

Could tweak that second sentence to something like:

Note that these options are categories and each one can be varied and tweaked in a variety of ways. Our aim here is to give a broad outline and distinguish the main categories from which one begins to assemble a business model. Think of it like a car: you choose the make and model and the broad features and then at the end you choose details like the specific color or trim or whether it has a sunroof or not.